packages/mdl-button/_variables.scss

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+:root {
+  --mdl-primary-color: rgb(63, 81, 181);
```

Comment:
Those var names are just placeholders until we get themeing laid out. I should add a TODO note on that...

On the note of sass variables, I don't see where they are adding any maintainability benefit here. What is the point of making each color into a sass var?

What do you mean by `primary/secondary/warn` buttons? Everything here is clearly used throughout the specification even though the exact details of what to use and when aren't. That's the fault of the spec team again though.